protoc-gen-swagger/genswagger/template.go

```diff
@@ -784,6 +784,17 @@ func applyTemplate(p param) (string, error) {
 					s.Info.Contact.Email = spb.Info.Contact.Email
 				}
 			}
+			if spb.Info.License != nil {
+				if s.Info.License == nil {
+					s.Info.License = &swaggerLicenseObject{}
```

Comment:
I think you need to add the `License` field to the `Info` object. I'm surprised this isn't a compile time error

Reply:
Shouldn't that be generated from "message Info" define in openapiv2.proto? Also when I build it with bazel it actually works. I am getting the license fields in the generated json.